mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -138,6 +138,12 @@ final class OneVsRestModel private[ml] (
     @Since("1.4.0") val models: Array[_ <: ClassificationModel[_, _]])
   extends Model[OneVsRestModel] with OneVsRestParams with MLWritable {
 
```

Comment:
Let's add a require() statement here which checks that models.nonEmpty is true (to throw an exception upon construction, rather than when numFeatures calls models.head below).  Just to be safe...